sbt-ash/src/main/scala/org/virtuslab/ash/AkkaSerializationHelperPlugin.scala

```diff
@@ -0,0 +1,89 @@
+        compilerPlugin(ashSerializabilityCheckerCompilerPlugin.value)),
+    Compile / scalacOptions ++= Seq(
+        s"-P:dump-persistence-schema-plugin:${(ashDumpPersistenceSchemaCompilerPlugin / ashCompilerPluginCacheDirectory).value}",
+        s"-P:registration-checker-plugin:${(ashRegistrationCheckerCompilerPlugin / ashCompilerPluginCacheDirectory).value}"),
+    cleanFiles ++= Seq(
+        (ashDumpPersistenceSchemaCompilerPlugin / ashCompilerPluginCacheDirectory).value / "dump-persistent-schema-cache",
+        (ashRegistrationCheckerCompilerPlugin / ashCompilerPluginCacheDirectory).value / "registration-checker-plugin-cache.csv"),
+    Compile / scalacOptions ++= (Compile / ashScalacOptions).value,
+    Test / scalacOptions --= (Compile / ashScalacOptions).value,
+    Test / scalacOptions ++= (Test / ashScalacOptions).value)
+
+  lazy val ashSettings: Seq[Def.Setting[_]] = baseAshSettings
+
+  lazy val baseAshSettings: Seq[Def.Setting[_]] = Seq(
+    ashDumpPersistenceSchemaCompilerPlugin := "org.virtuslab" %% "dump-persistence-schema-compiler-plugin" % (ashDumpPersistenceSchemaCompilerPlugin / ashCompilerPluginVersion).value,
+    ashRegistrationCheckerCompilerPlugin := "org.virtuslab" %% "registration-checker-compiler-plugin" % (ashRegistrationCheckerCompilerPlugin / ashCompilerPluginVersion).value,
+    ashSerializabilityCheckerCompilerPlugin := "org.virtuslab" %% "serializability-checker-compiler-plugin" % (ashSerializabilityCheckerCompilerPlugin / ashCompilerPluginVersion).value,
+    Compile / ashScalacOptions := {
+      val de = (Compile / ashDumpPersistenceSchemaCompilerPlugin / ashCompilerPluginEnable).value
+      val dv = (Compile / ashDumpPersistenceSchemaCompilerPlugin / ashCompilerPluginVerbose).value
+      val re = (Compile / ashRegistrationCheckerCompilerPlugin / ashCompilerPluginEnable).value
+      val rv = (Compile / ashRegistrationCheckerCompilerPlugin / ashCompilerPluginVerbose).value
+      val se = (Compile / ashSerializabilityCheckerCompilerPlugin / ashCompilerPluginEnable).value
+      val sv = (Compile / ashSerializabilityCheckerCompilerPlugin / ashCompilerPluginVerbose).value
+      compilerPluginFlagsToScalacOptions(de, dv, re, rv, se, sv)
+    },
+    Test / ashScalacOptions := {
+      val de = (Test / ashDumpPersistenceSchemaCompilerPlugin / ashCompilerPluginEnable).value
+      val dv = (Test / ashDumpPersistenceSchemaCompilerPlugin / ashCompilerPluginVerbose).value
+      val re = (Test / ashRegistrationCheckerCompilerPlugin / ashCompilerPluginEnable).value
+      val rv = (Test / ashRegistrationCheckerCompilerPlugin / ashCompilerPluginVerbose).value
+      val se = (Test / ashSerializabilityCheckerCompilerPlugin / ashCompilerPluginEnable).value
+      val sv = (Test / ashSerializabilityCheckerCompilerPlugin / ashCompilerPluginVerbose).value
+      compilerPluginFlagsToScalacOptions(de, dv, re, rv, se, sv)
+    },
+    ashCompilerPluginVersion := "0.2.0",
```

Comment:
Can this be un-hardcoded/point to the most recent released version by default? (how was it implemented so far?)